README.md

```diff
@@ -5,7 +5,34 @@
 The purpose of this API is to give users insight about events that are happening in Manitoba, without having to tread through the websites of many venues.
 
 ## Endpoints
+- ### [GET] Search all
+    ```https://what-is-good-manitoba.ca/```
+
+- ### [GET] Search based on area and datetime range
+    ```https://what-is-good-manitoba.ca?location=area&start=datetime&end=datetime```
+
+  <br>
+
+    #### Parameters: 
+    - area: the name of the city
+    - start: *numeric values* formatted as year/month/date/hour
+    - end: *numeric values* formatted as year/month/date/hour
+
+<br>
+
+- ### [POST] Upload new event
+    ```https://what-is-good-manitoba.ca/submit```
+
+  <br>
+
+    #### Body (please see resources): 
+    - event
+    - performer
+    - venue
+
+<br>
 
 ## Resources
 
-## Samples
+## Sample requests and responses
```

Comment:
I think we only need one sample so change this to singular (but also Dirk should be working on this so you could remove it/not do anything)